client/src/ThingsExplorer/ItemsProvider.ts

```diff
@@ -32,7 +46,7 @@ const THING_TEMPLATE = (thing: Thing): SnippetString => {
         let channels = _(thing.channels)
             .filter(channel => channel.kind === 'STATE')
             .map((channel: Channel) => {
-                let name = s.classify(thing.label) + '_' + s.classify(channel.id);
+                let name = generateItemName(thing.label) + '_' + s.classify(channel.id);

```

Comment:
Shouldnt this '_' also be part of the casing?

CamelCase will be BedroomLight_Switch and Bedroom_Light_Switch for snake case. This seems to be inconsistent in my opinion.

Reply:
@SamuelBrucksch 

> CamelCase will be BedroomLight_Switch

that's just the way it was before. I was assume this was designed on purpose, to distuingish between the `Thing` and `Channel`?

> and Bedroom_Light_Switch for snake case

this would be consistent, as all is snake case here :wink:

Reply:
Sure the second one is right, the first one is mixed. I checked the Readme and yes, it was like that before. @Confectrician, @kubawolanin what do you think?

BedroomLightSwitch or BedrommLight_Switch for camel case?

Reply:
I am not a friend of this mixed version to be honest.
It seems inconsistent too in my view and as we are changing something in that code area anyway i would vote for "real Camelcase" without the underscore.